jdaviz/core/template_mixin.py

```diff
@@ -2899,16 +2899,19 @@ def _on_viewer_layer_changed(self, msg=None):
                 if glue_name in ['contour_visible', 'bitmap_visible']:
                     state.add_callback('visible', self._on_glue_layer_visible_changed)
 
-                if self.sync.get('choices') is None and \
-                        (hasattr(getattr(type(state), glue_name), 'get_display_func')
-                         or glue_name == 'cmap'):
+                if (
+                    self.sync.get('choices') is None and
+                    (
+                        hasattr(getattr(type(state), glue_name), 'get_display_func') or
+                        glue_name == 'cmap'
+                    )
+                ) or glue_name == 'cmap_att':
```

Comment:
does this need to be outside the choices is None logic because the choices are changing (and so the comment below doesn't really apply?) or can we just have `glue_name in ('cmap', 'cmap_att')`?

Reply:
I had wondered the same thing! But `glue_name in ('cmap', 'cmap_att')` didn't work, so I think the answer to your first question is yes.

Reply:
ok, let's just update the comment a bit then and call this good!

Reply:
Comments updated.